### ISSUE TITLE
App wont load data

### ISSUE THREAD

**Issue by digivi13** [Dacredible/stock-search#2]
This is similar to the last issue. I believe your amazon credits have run out and I would like to build my own backend to process the alpha vantage. Can you post the code you used?

**Comment by Dacredible** [comment]
yes I've already shutdown the backend, the source code should in my repo for csci571, I can't remember  exactly which one.

**Comment by digivi13** [comment]
Thanks for the reply. Correct me if i'm wrong, but I believe this is the source code you are referring to:
https://github.com/Dacredible/CSci571/tree/master/public_html/hw8/stock-search/backend

**Comment by digivi13** [comment]
If/when I successfully get the backend hosted on AWS I will post it here so the app should continue to be functional.

**Comment by Dacredible** [comment]
> If/when I successfully get the backend hosted on AWS I will post it here so the app should continue to be functional.

sure, thanks

**Comment by Dacredible** [comment]
> Thanks for the reply. Correct me if i'm wrong, but I believe this is the source code you are referring to:
> https://github.com/Dacredible/CSci571/tree/master/public_html/hw8/stock-search/backend

Yes it is

**Comment by Dacredible** [comment]
I mean you can even try to edit the app code and change it to your backend, then submit a pull request. Are you doing the same project here?

**Comment by digivi13** [comment]
I will probably end up doing that. I intend to build upon your project, allowing for users to build xgboost and other types of machine learning predictive models for whatever stock they choose, and monitor how those models perform over time.

**Comment by digivi13** [comment]
Using shap values you can do some pretty cool stuff. One of the ideas I am experimenting with is building xgboost models, fitting them to data, then using shap values to rank the impact of different indicators on predictions.

**Comment by digivi13** [comment]
Just curious, do you have any good resources for deploying to amazon elastic beanstalk?

**Comment by digivi13** [comment]
alright so turns out that was not the link where the backend ios files were stored. I found them here: 
https://github.com/Dacredible/CSci571/tree/master/public_html/hw8/stock-search/backend%20copy

I changed the API key in all instances in the app.js backend file to the one I recieved after signing up for alpha vantage, then re-zipped and deployed on amazon elastic beanstalk which was wicked easy. I also changed the base URL in the xcode project to the following: 
http://stockpredict-env-1.kmxzra3mdh.us-west-2.elasticbeanstalk.com/api/ios/

This link is now live, although it may be down from time to time as I make modifications to the application.
